ts/src/phemex.ts

```diff
@@ -159,6 +159,7 @@ export default class phemex extends Exchange {
                 },
                 'v2': {
                     'get': {
+                        'public/products': 5,
```

Comment:
I'm afraid we lost some markets here, previously we had 808 and now 781
![image](https://github.com/ccxt/ccxt/assets/43336371/489e6c6e-539a-4408-935f-17e8cdcfd4d0)

Reply:
On the other hand, we have some of the missing markets as well (for instance BTC/USDC was not available previously)
![image](https://github.com/ccxt/ccxt/assets/43336371/de3ac0ec-7078-45c7-9f2b-54735fd49969)

Reply:
that's what I told in https://github.com/ccxt/ccxt/issues/20230.
You can check, that on web https://phemex.com/spot/trade/ there is no SWEAT/USDT, XTP/USDT, INS/USDC and some other pairs (at least spot), also there is no any orderbook for this pairs

Reply:
@ndubel Yeah you;'re right, these are the ones missing but I can't find them in the platform
![image](https://github.com/ccxt/ccxt/assets/43336371/58f1f2e4-1f0f-4f6a-bbfc-f37e7f113eed)